ffi/capi/bindings/js/ICU4XWeekCalculator.mjs

```diff
@@ -2,6 +2,7 @@ import wasm from "./diplomat-wasm.mjs"
 import * as diplomatRuntime from "./diplomat-runtime.mjs"
 import { ICU4XError_js_to_rust, ICU4XError_rust_to_js } from "./ICU4XError.mjs"
 import { ICU4XIsoWeekday_js_to_rust, ICU4XIsoWeekday_rust_to_js } from "./ICU4XIsoWeekday.mjs"
+import { ICU4XWeekendContainsDay } from "./ICU4XWeekendContainsDay.mjs"
```

Comment:
Issue: did you git add `ICU4XWeekendContainsDay.mjs`?

Reply:
I did now.